### DISCUSSION TITLE
Laid off from job and lost access to that Github account while still subscribed to Copilot

### DISCUSSION THREAD

**Discussion by AndyKChan** [community/community#70346] [New to GitHub]
So I got laid off by my previous company while using a company Github account. I lost access to my work laptop almost instantly and I had forgotten that I had subscribed to the 30 day trial using Github Copilot. So the IT guy at my previous company is assisting me with this because he has access to the work email associated to the Github account. Using my user login credentials and the work email, I was still not able to login to my account, so I had to make a Github support request to either delete my account or help me unsubscribe to Copilot.

It's now been about 10 days since I've sent in the support request which was about 5 days before the trial period ended and I still have not gotten a response for Github support. I would really love to get this done with and be unsubscribed to Copilot on an account that nobody has access to.

How long does it usually take support to respond? I've also told Github support to email me through my personal email where I have access immediately instead of asking my IT guy if I've received anything yet everyday, will they actually do that for me?

TLDR:
I can't get access to my Github account that's subscribed to copilot and support won't respond.

**Comment by queenofcorgis** [comment]
Hi @AndyKChan, 

Thank you for your patience and I so sorry to hear you were laid off. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
